### PR TITLE
Add Initial Issue Template

### PR DESCRIPTION
**Pull request by rmccorm4**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Operating System + Version**: 
+**GPU Type**: 
+**Nvidia Driver Version**: 
+**CUDA Version**: 
+**CUDNN Version**: 
+**Python Version (if applicable)**: 
+**TensorFlow Version (if applicable)**: 
+**PyTorch Version (if applicable)**: 
+**TensorRT Version**:
```
Comment (rajeevsrao) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Please move TRT version to the top.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+**PyTorch Version (if applicable)**: 
+**TensorRT Version**: 
+**Baremetal or Container (if container which image + tag)**: 
+
+
+## Steps To Reproduce
+
+Craft a minimal bug report following this guide - https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports
+
+This helps us reproduce the issue you're having and resolve the issue more quickly:
```
Comment (rajeevsrao) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Request for build command and steps to run the test/usecase. Attach or links to any models or relevant data files.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (rmccorm4) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (rmccorm4) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

**Pull request merged**